test/runtests.jl

```diff
@@ -234,6 +257,12 @@ const BT = BinaryTrees
     @inferred BT.insert!(tree, 2)
     @inferred BT.insert!(tree, 1)
     @inferred BT.insert!(tree, 3)
+    @inferred Nothing BT.minnode(tree)
+    @inferred Nothing BT.maxnode(tree)
+    @inferred Tuple{Union{BT.AVLNode,Nothing},Union{BT.AVLNode,Nothing}} BT.prevnext(
+      tree,
+      2
+    )
```

Comment:
Maybe this will work. @souma4, can you try?
```suggestion
    @inferred Nothing BT.prevnext(tree, 2)[1]
    @inferred Nothing BT.prevnext(tree, 2)[2]
```

Reply:
That does work. Switched those over.